src/sql/workbench/contrib/queryplan2/browser/widgets/customZoomWidget.ts

```diff
@@ -37,7 +37,7 @@ export class CustomZoomWidget extends QueryPlanWidgetBase {
 		});
 		attachInputBoxStyler(this.customZoomInputBox, this.themeService);
 
-		const currentZoom = queryPlanView.azdataGraphDiagram.graph.view.getScale();
+		const currentZoom = queryPlanView.azdataGraphDiagram.graph.view.getScale() * 100;
```

Comment:
is the input restricted to numbers? I'd recommend having some predefined values for user to select. if not let's add a '%' after the input to make it ovbvious.

Reply:
It populates automatically with current zoom level of the graph.